setup.py

```diff
@@ -88,7 +88,7 @@ def get_git_sha():
         "isodate",
         "markdown>=3.0",
         "msgpack>=0.6.1, <0.7.0",
-        "pandas>=0.25.3, <1.0",
+        "pandas>=1.0.3, <1.1",
```

Comment:
This feature requires functionality that was only recently introduced in Pandas 1.0.